jalib/jalloc.cpp

```diff
@@ -28,6 +28,8 @@
 #include <string.h>
 #include <unistd.h>
 
+#define ATOMIC_SHARED volatile __attribute((aligned))
```

Comment:
Can we put this and other macros in dmtcp.h near https://github.com/dmtcp/dmtcp/blob/master/include/dmtcp.h#L51 ? This would allow other files to use it as needed.

Reply:
Done.